compiler/rustc_const_eval/src/const_eval/fn_queries.rs

```diff
@@ -15,20 +15,14 @@ fn parent_impl_constness(tcx: TyCtxt<'_>, def_id: LocalDefId) -> hir::Constness
     }
 }
 
-/// Checks whether an item is considered to be `const`. If it is a constructor, it is const.
-/// If it is an assoc method or function,
-/// return if it has a `const` modifier. If it is an intrinsic, report whether said intrinsic
-/// has a `rustc_const_{un,}stable` attribute. Otherwise, panic.
```

Comment:
Having an explicit list of which function-like items are considered "const" seems useful. It doesn't have to be in the doc comment, maybe it should go inside this function, but why remove it entirely?

Reply:
It's on the query

Reply:
[    /// [^1]: Tuple struct/variant constructors, closures and free, associated and foreign functions.](https://github.com/rust-lang/rust/blob/8038371bf4b3b34f90fabc524d9eec87f4d942f1/compiler/rustc_middle/src/query/mod.rs#L792)